### PR TITLE
Replace bootstrap with Zendesk Garden css

### PR DESCRIPTION
**Pull request by adammw**
:v:

/cc @zendesk/vegemite @zendesk/zdbreaking 

### Description

Replace Bootstrap with Zendesk Garden CSS.

Although this is a breaking change, the idea of how developers consume this code is that they create a copy of it, as such this should only affect new developers creating new apps (hence the from-scratch branch).

screen shot 2017-01-04 at 4 41 56 pm

### References
* JIRA: https://zendesk.atlassian.net/browse/AF-452

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -7,10 +7,10 @@ var HtmlWebpackPlugin = require('html-webpack-plugin');
 
 var externalAssets = {
   css: [
-    'https://cdn.jsdelivr.net/bootstrap/3.3.7/css/bootstrap.min.css'
+    'https://assets.zd-staging.com/apps/sdk-assets/css/0/zendesk_garden.css'
   ],
   js: [
-    'https://cdn.jsdelivr.net/g/lodash@4.14.0,handlebarsjs@4.0.5,jquery@3.1.0,bootstrap@3.3.7,momentjs@2.14.1',
```
Comment (danielbreves) [inline webpack.config.js]: Why remove moment?

**Comment by danielbreves** [comment]
👍  once this is targeting production.

File: webpack.config.js (hunk 1 above)
Comment (adammw) [inline webpack.config.js]: It's from scratch, do we really need it? It's not used anywhere in the demo code...

File: src/stylesheets/app.scss [hunk 2]
```diff
@@ -1 +1,5 @@
 @import './ticket_sidebar.scss';
+
+body {
+  margin: 0;
```
Comment (adammw) [inline src/stylesheets/app.scss]: this is to remove the default margin present in the browser stylesheet and fit in the ticket sidebar. we may want to make this conditional, ie. only for sidebar apps, but I was hoping that developers could just delete/change this if that's needed.

**Pull request merged**